lib/data/multipleBackendGateway.js

```diff
@@ -199,6 +199,21 @@ const multipleBackendGateway = {
         }
         return cb();
     },
+
+    objectTagging: (method, key, bucket, objectMD, log, cb) => {
+        const location = objectMD.location[0].dataStoreName;
```

Comment:
this won't work for 0 byte objects.  we need to know from the metadata of the object, what the data backend is.